spec/features/projects/show_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe 'projects show ', type: :feature do
+  describe 'as a user' do
+    let(:project) do
+      recycled_material_challenge = Challenge.create!(theme: 'Recycled Material', project_budget: 1000)
+      recycled_material_challenge.projects.create!(name: 'News Chic', material: 'Newspaper')
+    end
```

Comment:
I like the use of let! Can you explain the difference in using let vs before(:each)? You don't need to tell me, but it is a question that comes up. :)